feed.rb

```diff
@@ -0,0 +1,38 @@
+require 'rss'
+require 'rss/2.0'
+require 'open-uri'
+require 'fileutils'
+require 'safe_yaml'
+require 'httparty'
+require 'feedjira'
+
+url = 'https://s8hdp1sdz5.execute-api.eu-west-2.amazonaws.com/latest/proxy/'
```

Comment:
What url is this? Did I make this proxy or you? Acast has it's own RSS feed url. Can't we use that? `feed:https://feeds.acast.com/public/shows/6012fcea40e3fd5f183effd9`

Reply:
Don't know where it came from - updated in latest version